controllers/bsl.go

```diff
@@ -145,6 +156,23 @@ func (r *DPAReconciler) ReconcileBackupStorageLocations(log logr.Logger) (bool,
 	return true, nil
 }
 
+func (r *DPAReconciler) UpdateLabels(secretName string, namespace string, dpaName string) (bool, error) {
```

Comment:
NIT: can we rename to `UpdateCredentialsSecretLabels` just to be explicit?

Reply:
done